modules/tracing/mule-tracer-configuration-impl/src/main/java/module-info.java

```diff
@@ -30,4 +32,6 @@
       org.mule.runtime.spring.config,
       spring.beans;
 
+  opens org.mule.runtime.tracing.level.impl.config to spring.core;
```

Comment:
Why is this needed?

Reply:
This seems to be needed for the tests in java 17.

`Error creating bean with name '_muleCoreComponentTracerFactory': Unsatisfied dependency expressed through field 'initialSpanInfoProvider'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name '_muleTracerInitialSpanInfoProvider': Unsatisfied dependency expressed through method 'setTracingLevelConfiguration' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name '_muleTracingLevelConfiguration': Injection of autowired dependencies failed; nested exception is java.lang.reflect.InaccessibleObjectException: Unable to make field private org.mule.runtime.core.api.MuleContext org.mule.runtime.tracing.level.impl.config.AutoConfigurableTracingLevelConfiguration.muleContext accessible: module org.mule.runtime.tracer.configuration.impl does not "opens org.mule.runtime.tracing.level.impl.config" to module spring.core`